backend/demo2.py

```diff
@@ -0,0 +1,29 @@
+import base64
+import json
+
+# Data you want to share
+data = {
```

Comment:
are these demos gonna stay in the repo?

Reply:
No, this is only for testing purposes. Before our final publish, we will delete the demo files